satellite/payments/stripecoinpayments/database.go

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package stripecoinpayments
+
+import (
+	"context"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+)
+
+// Customers is interface for working with stripe customers table
+type Customers interface {
```

Comment:
I would rename this interface to `CustomersDB`, it would make more clear what is.

Reply:
Shouldn't this interface be generic to all the payment providers?

Reply:
we will have some kind of PaymentDB, but now lockgen could not handle nested dependencies.

In future we will have smth like:
```
type PaymentDB interface {
    Invoices()
    Customers()
    Transactions()
}
```